### PR TITLE
Added username support in HUBOT_AUTH_ADMIN var

### PR DESCRIPTION
**Pull request by sebastiandero**
Check the issue description for more information, this adds username support to hubot auth admin env_var

### CODE REVIEW

File: src/auth.coffee [hunk 1]
```diff
@@ -38,7 +38,7 @@ module.exports = (robot) ->
 
   class Auth
     isAdmin: (user) ->
-      user.id.toString() in admins
+      (user.id.toString() in admins) or (user.name.toString() in admins)
```
Comment (sebastiandero) [inline src/auth.coffee]: This allows for either id or name to be in the ENV_VAR

**Comment by sebastiandero** [comment]
Issue: #48

**Comment by nick-woodward** [comment]
Hey @sebastiandero.

Unfortunately the reason this functionality has never been implemented 
in the past was due to the ability to fake usernames in some (if not all) 
of the clients supported by hubot. 

This means that usernames aren't a secure way of ensuring that the user 
making a request is indeed an admin.

**Comment by sebastiandero** [comment]
@ceci-woodward I see, maybe a 'Hubot get my id' command would be nice, I have had to implement this in my script since in slack there is no other way for a user to get their ID

**Comment by nick-woodward** [comment]
@sebastiandero Yeah, the only other way I know of to get your id is via 
the slack api or logging it in the console which is a bit of a pain.

If you have the time, feel free to modify this PR to add a `whats my id?` 
command and I'll look it over.

**Comment by sebastiandero** [comment]
@ceci-woodward will do this week!

**Pull request closed without merging**